Mathlib/FieldTheory/PolynomialGaloisGroup.lean

```diff
@@ -198,6 +198,14 @@ instance galAction [Fact (p.Splits (algebraMap F E))] : MulAction p.Gal (rootSet
     simp only [smul_def, Equiv.apply_symm_apply, Equiv.symm_apply_apply, mul_smul]
 #align polynomial.gal.gal_action Polynomial.Gal.galAction
 
+lemma galAction_isPretransitive [Fact (p.Splits (algebraMap F E))] (hp : Irreducible p) :
```

Comment:
I wonder how hard it would be to upgrade this to an iff.

Reply:
It's not an `iff` (e.g., `x^2`, or more generally `p^2` for any irreducible polynomial `p`).

Reply:
Ah indeed. It needs separability conditions.

Reply:
Inseparability is fine, we just need square-freeness.